samples/sensor/bme68x/README.rst

```diff
@@ -0,0 +1,48 @@
+.. _bme68x:
+
```

Comment:
The sample is now placed in the "Other samples" section. Since now, we have two sensor samples:  BH1749: Ambient Light Sensor IC and this sample. We could have a new sample category : Sensor samples in docs.

Reply:
Sure! How do we do this?